components/automate-deployment/pkg/server/automate_config.go

```diff
@@ -409,3 +438,14 @@ func configLogrotate(req *config.Log) error {
 func rollbackLogrotate() error {
 	return os.Remove(logRotateConfigFile)
 }
+
+func UpdateOfLogroateConfig(req *api.PatchAutomateConfigRequest, existingCopy *deployment.AutomateConfig) (*api.PatchAutomateConfigRequest, error) {
+	if req.GetConfig().GetGlobal().GetV1().GetLog().GetRedirectSysLog().GetValue() == true && existingCopy.GetGlobal().GetV1().GetLog().GetRedirectSysLog().GetValue() == true {
+		if err := mergo.Merge(req.Config, existingCopy); err != nil {
```

Comment:
Have we tested this, if we are getting default values in the req, its getting overrided?